src/Magnum/Math/Functions.h

```diff
@@ -410,6 +411,33 @@ template<std::size_t size, class T> inline Vector<size, T> ceil(const Vector<siz
     return out;
 }
 
+
+/**
+@brief Computes a Binomial Coefficient
```

Comment:
```suggestion
@brief [Binomial Coefficient](https://en.wikipedia.org/wiki/Binomial_coefficient)
```

for people like me, who would need to google this anyway :laughing: